### ISSUE TITLE
BigInteger interpretation 

### ISSUE THREAD

**Issue by MeskaKeyne** [usebruno/bruno#1402]
Hello

it seems that the BigInteger are misinterpreted by Bruno

In postman it works

**Comment by Its-treason** [comment]
Do you mean in a JSON response, like in #869 or in vars? Like in #1000

**Comment by MeskaKeyne** [comment]
yes its same problem

**Comment by helloanoop** [comment]
Yes, in case of bigint. This happens due to the typecasting process.
The digits get lost during serialisation and deserialisation process.
This a bug and needs to be fixed.

Also, There is a standardization effort underway for [bru lang](https://github.com/brulang/bru-lang)
I am taking care of this edge case in the parser as seen here: https://github.com/brulang/bru-js/blob/main/src/parse.js#L52-L54

Will get this fixed as soon as I can in the current version of bru lang.

cc @halostatue

**Comment by Raffine64** [comment]
@helloanoop do you have an idea when this could be addressed ?

in the latest version we still get the problem (original correct number: 1000000000000108012123):
  {
      "id": 1.0000000000001075e+21,
...

**Comment by helloanoop** [comment]
This has been fixed in [PR](https://github.com/usebruno/bruno/pull/6026) and is live in `v2.14.0` 🎉 🎉
